### DISCUSSION TITLE
Chaining query for many-to-many relationships

### DISCUSSION THREAD

**Discussion by axhamre** [prisma/prisma#13391] [Q&A] [answered]
In ActiveRecord I'm used to be able to use fluent interface queries on a many-to-many relationship, something similar to this:

`await db.user.findUnique({ where: { id: 1 } }).workspaces()`

Is this possible to accomplish somehow with Prisma? Condensed schema.prisma:

```
model Workspace {
  id         String       @id @default(uuid())
  Membership Membership[]
}

model User {
  id         Int          @id @default(autoincrement())
  firstName  String
  lastName   String
  Membership Membership[]
}

model Membership {
  workspace   Workspace @relation(fields: [workspaceId], references: [id])
  workspaceId String
  user        User      @relation(fields: [userId], references: [id])
  userId      Int

  @@id([workspaceId, userId])
}
```

**Comment by richardwardza** [comment, marked as answer]
Hi @axhamre 
What you're looking for could be along the lines of:
```
await db.user.findUnique({ where: { id: 1 }, include: {
  Membership: {
    include: {
      workspace: true
    }
  }
} }).workspaces()
```

Check this for more though: https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#nested-reads

**Reply by axhamre** [reply]
Thank you!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
